build.cmd

```diff
@@ -1,7 +1,7 @@
 @echo off
 setlocal
 
-set MSBUILD_ARGS="%~dp0build.proj" /verbosity:minimal /fileloggerparameters:Verbosity=diag;LogFile="%~dp0msbuild.log" %*
```

Comment:
Because I conflicted with myself during rebase and I solved the conflict wrong :)